From: Simon McVittie <smcv@debian.org>
Date: Sat, 11 Jun 2016 19:07:40 +0100
Subject: test-sysroot.js: set "strict mode" when sourcing libtest.sh

As with the C tests in commit 08580118, this makes sure the test
fails as soon as something goes wrong.

Signed-off-by: Simon McVittie <smcv@debian.org>
Forwarded: https://github.com/ostreedev/ostree/pull/335
Applied-upstream: 2016.6, commit:0cf0fa815414eebb3e642429085fa75dda05c24d
---
 tests/test-sysroot.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test-sysroot.js b/tests/test-sysroot.js
index 9468d2f..7c31659 100644
--- a/tests/test-sysroot.js
+++ b/tests/test-sysroot.js
@@ -37,7 +37,7 @@ function libtestExec(shellCode) {
     let testdatadir = GLib.getenv("G_TEST_SRCDIR");
     let libtestPath = GLib.build_filenamev([testdatadir, 'libtest.sh'])
     let proc = GSystem.Subprocess.new_simple_argv(['bash', '-c',
-						   '. ' + GLib.shell_quote(libtestPath) + '; ' + shellCode],
+						   'set -xeuo pipefail; . ' + GLib.shell_quote(libtestPath) + '; ' + shellCode],
 						  GSystem.SubprocessStreamDisposition.INHERIT,
 						  GSystem.SubprocessStreamDisposition.INHERIT,
 						  null);
